systemtests/util_test.go

```diff
@@ -31,7 +32,11 @@ func (s *systemtestSuite) readIntent(fn string) (*config.TenantConfig, error) {
 		return nil, err
 	}
 
-	cfg := &config.TenantConfig{}
+	cfg := &config.TenantConfig{
```

Comment:
yes, we can do this.

Or see my next comment, may using `config.NewDefaultTenantConfig()` here as well might be better as that will help us test that `ceph` is picked as default driver when one is not specified by the user.